stake-pool/program/src/stake.rs

```diff
@@ -102,6 +103,55 @@ pub enum StakeInstruction {
     AuthorizeWithSeedNOTUSED,
 }
 
+/// FIXME copied from the stake program
```

Comment:
I hope that all of this can get removed soon :-)

Reply:
🙏